doc/modules/preprocessing.rst

```diff
@@ -455,7 +455,7 @@ K-bins discretization
   >>> X = np.array([[ -3., 5., 15 ],
   ...               [  0., 6., 14 ],
   ...               [  6., 3., 11 ]])
-  >>> est = preprocessing.KBinsDiscretizer(n_bins=[3, 3, 2]).fit(X)
+  >>> est = preprocessing.KBinsDiscretizer(n_bins=[3, 3, 2], encode='ordinal').fit(X)
```

Comment:
Below please note that by default the output is one-hot encoded into a sparse matrix (use `:ref:` to link to the right place), but that this can be configured with the `encode` parameter.